zerver/tests/fixtures/katex_test_cases.json

```diff
@@ -0,0 +1,36 @@
+            "expected_output": "intro text\n\n```math\na + b = 33\n\n\n\n\n\n\n\n\nd+x = 44\n``` \n\nend"
```

Comment:
Even if it's not used for anything, I think an `original_markup` field that has what you put into the compose box to generate the original message would be very helpful. Ideally, we'd assert that `original_input`, after `markdown.ts` rendering, generates `input`, and cases where that didn't equal `expected_output` would be a nice way to document any know test cases where the translation isn't perfect. (Say, the whitespace changes slightly).